src/main/java/net/minestom/server/collision/EntityCollision.java

```diff
@@ -26,7 +26,7 @@ final class EntityCollision {
             SweepResult sweepResult = new SweepResult(minimumRes, 0, 0, 0, null, 0, 0, 0, 0, 0, 0);
 
             if (!entityFilter.apply(e)) continue;
-            if (!e.hasCollision()) continue;
+            if (!e.isCollidesWithEntities()) continue;
```

Comment:
`hasEntityCollisions` would be a better name

Reply:
done